internal/pkg/gateway/commit/blockparser.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2021 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package commit
+
+import (
+	"github.com/golang/protobuf/proto"
+	commonProto "github.com/hyperledger/fabric-protos-go/common"
+	peerProto "github.com/hyperledger/fabric-protos-go/peer"
```

Comment:
Thank you for the link. I haven't read that blog before. I was personally finding my reading of this code easier when I could see at a glance where I was working with protobufs rather than one of the other Fabric packages with the same name but I appreciate this is not idiomatic so I'll correct it